### PR TITLE
(maint) alphabetise lists and add new modules

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

**Comment by tphoney** [comment]
@Iristyle @glennsarti @DavidS  hopefully not controversial. Bar how you spell alphabetize 😺

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,6 @@ language: ruby
 rvm: 2.3.1
 cache: bundler
 # Run msync but in noop (Test Command).  Also, do not update the private modules (regex escaped for yaml)
-script: 'bundle exec msync update --noop --git-base=https://github.com/ -x "(sqlserver|vsphere|yang_ietf|satellite_pe_tools|websphere_application_server)"'
+script: 'bundle exec msync update --noop --git-base=https://github.com/ -x "(satellite_pe_tools|sqlserver|vsphere|yang_ietf|yang_openconfig|websphere_application_server)"'
```
Comment (eputnam) [inline .travis.yml]: might as well also move websphere while you're at it

Comment (tphoney) [reply .travis.yml]: ahh alphabets. still tricky for me

**Comment by tphoney** [comment]
@glennsarti @DavidS  can i get a merge please

**Pull request merged**